app/cells/plugins/core/content_item/popup.haml

```diff
@@ -3,14 +3,17 @@
 
 = render_label
 %br
-%button.content_item_button.text-center.popup--open
-  .button_content
+%button.content-item-button.text-center.popup--open
```

Comment:
since you've applied centering to `.content-item-button` you can remove `.text-center`